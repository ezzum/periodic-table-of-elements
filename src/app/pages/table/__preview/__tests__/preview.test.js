import { mount } from 'enzyme';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import Preview from '../preview';

describe('Preview tests', () => {
  const initialState = {
    previewElement: {
      number: 3,
      name: 'литий',
      group: 'metal',
      symbol: 'Li',
      atomicMass: 1.023,
    },
  };
  const mockStore = configureStore();
  let store, wrapper;

  beforeEach(() => {
    store = mockStore(initialState);
    wrapper = mount(
      <Provider store={store}>
        <Preview />
      </Provider>
    );
    console.log(wrapper.debug());
  });

  it('snapshot test', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
