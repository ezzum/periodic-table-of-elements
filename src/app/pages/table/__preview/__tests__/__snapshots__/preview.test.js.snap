// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preview tests snapshot test 1`] = `ReactWrapper {}`;
